zerver/tests/test_realm.py

```diff
@@ -1759,7 +1760,7 @@ def do_test_changing_settings_by_owners_only(self, setting_name: str) -> None:
             vals = bool_tests
         assert vals is not None
 
-        self.set_up_db(setting_name, vals[0])
+        do_set_realm_property(get_realm("zulip"), setting_name, vals[0], acting_user=None)
```

Comment:
Similar to above, this seemed better than modifying the lists above to have default value as the first value in the list in `vals`.